pkgs/development/python-modules/nix-prefetch-github/default.nix

```diff
@@ -15,11 +15,11 @@
 
 buildPythonPackage rec {
   pname = "nix-prefetch-github";
-  version = "4.0";
+  version = "4.0.1";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "sha256-STUyMUCWAHfDA6dkpiOqSRBL3/tubedUbWa94Kp/764=";
+    sha256 = "sha256-asfRohjOgYxMV/wprKvxUD328GVJQkAYnuAkE09kKgs=";
```

Comment:
```suggestion
    sha256 = "asfRohjOgYxMV/wprKvxUD328GVJQkAYnuAkE09kKgs=";
```

Reply:
What is the advantage of this?

Reply:
Why should sha256 be there twice?

Reply:
Why should sha256 not be there twice?

Reply:
I feel like you are trolling me. Don't appreciate that.

Reply:
@SuperSandro2000 Can you tell me what your problem is? I asked you the simple question what advantage of your change is. I asked because it is not clear to me. You answered my question with another question which does not make any sense to me. And after expressing that I feel disrespected by you I get a thumbs down. Now I feel stupid and disrespected.

Reply:
@seppeljordan I believe that @SuperSandro2000's question was rhetorical, they are saying that repeating "sha256" is unnecessary.

Reply:
@expipiplus1 Understood. And don't get me wrong, I can put this in the PR no problem, but...

You see I was asking since the program in question does calculate checksums for github repositories intended to be used with nix. And currently it outputs hashes in the same formatting as the one in question in this PR. If the redundant 'sha256-' is the main problem then I don't really care too much. Should it be a standard for this repo that "we don't put sha256- in front of our hashes" then I would very much care. There could be other issues with this format that I wouldn't know about, e.g. compatibility issues with external tools.

I am not expecting an answer, but should you know about any of this it would be awesome if you would tell me.

Reply:
I don't know if it's a standard, but `sha256 = "sha256-` only appears 199 times in the whole of nixpkgs, so it seems to be a more or less de-facto standard. Perhaps this should be taken up against the nix-prefetch-github repo.

In terms of compatibility I don't know off the top of my head if any other tools will break, but other tools (like `update-nix-fetchgit`, and the version of `nix-prefetch-github` in the current nixpkgs HEAD do not output the `sha256-` prefix.